syntax = "proto3";

package viam.service.summation.v1;

import "google/api/annotations.proto";

service SummationService {
  rpc Sum(SumRequest) returns (SumResponse) {
    option (google.api.http) = {
      post: "/viam/api/v1/service/summation/{name}/sum"
    };
  }
}

message SumRequest {
  string name = 1;
  repeated double numbers = 2;
}

message SumResponse {
  double sum = 1;
}
